java-client/pom.xml

```diff
@@ -127,13 +127,13 @@
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-context</artifactId>
-            <version>${spring.version}</version>
```

Comment:
Please double check if this rename could somehow downgrade spring dependency for some of agents. We had such situation with CEP Esper in the past.

```
$ grep . -rnw -e "spring.version"
./lwm2m-agent/lwm2m-decoders/api/pom.xml:43:            <version>${spring.version}</version>
./lwm2m-agent/lwm2m-decoders/api/pom.xml:48:            <version>${spring.version}</version>
./lwm2m-agent/lwm2m-integration-tests/pom.xml:25:        <spring.version>4.3.20.RELEASE</spring.version>
```

Reply:
Added backward compatibility property rewrite to allow for independent change in agents and microservices.